sort.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$(readlink -e "$(dirname "${0}")")
+
+# Install jq
+if ! (command -v jq > /dev/null 2>&1); then
+    sudo apt-get install -y jq
+fi
+
+# Set target file
+cspell_json="$SCRIPT_DIR/cspell/.cspell.json"
+
+# Create new json
+sorted_json=$(jq --indent 4 \
```

Comment:
Question: `.editorconfig` defines `indent_size = 2`. Is the `jq --indent 4` no problem with `.editorconfig`?

Reply:
`indent_size = 2` is only for `.yml`(GitHub actions).

Reply:
Oh, yes, the setting is not for `.json`. Sorry, I just misunderstood.